api/tests/State/Util/AbstractPersistProcessorTest.php

```diff
@@ -42,6 +42,20 @@ protected function setUp(): void {
         );
 
         $this->processor->method('onBefore')->willReturnArgument(0);
+
+        set_error_handler(
+            /**
+             * @throws WarningException
+             */
+            static function (int $errno, string $errstr): never {
```

Comment:
What is this "never"? Never seen that before

Reply:
I copied it and did not check what it does.
Now i did: https://www.php.net/manual/en/language.types.never.php